snuba/writer.py

```diff
@@ -10,7 +11,7 @@
 from snuba.clickhouse import DATETIME_FORMAT, Array
 
 
-logger = logging.getLogger('snuba.writer')
+logger = logging.getLogger("snuba.writer")
```

Comment:
are we converging on double quotes as a standard?

Reply:
I just did whatever black told me to do (since we're using that in other projects, and I've used on a few files here, and think we should start migrating over to): https://github.com/psf/black#strings